src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,25 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int high;
+
     public Excavator() {
     }
 
+    public Excavator(String name, String color, int high) {
+        this.name = name;
+        this.color = color;
```

Comment:
```suggestion
        super(name, color); 
```
this will set these fields, but you shouldn't make them protected

Reply:
good idea
thank you